packages/build/src/packaging/package/debian.ts

```diff
@@ -38,7 +38,7 @@ export async function createDebianPackage(
 
   if (pkg.manpage) {
     // Put manpage file in /usr/share/man/man1/.
-    const manualDir = path.join(dir, pkg.metadata.debName, 'usr', 'share', 'man', 'man1');
+    const manualDir = path.join(dir, pkg.metadata.debName, 'usr', 'share', 'man', 'man' + getManSection(pkg.manpage.packagedFilePath));
```

Comment:
We hardcoded the manual section here without really needing to, not a bad thing in any way but since I was here I thought I’d make this a bit more generic as well